package.json

```diff
@@ -29,13 +29,13 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "babel-polyfill": "^6.23.0",
-    "babel-runtime": "^6.23.0",
+    "babel-runtime": "^6.26.0",
     "isomorphic-fetch": "^2.2.1",
-    "lodash.merge": "^4.6.0",
+    "lodash.camelcase": "^4.3.0",
+    "lodash.merge": "^4.6.2",
     "lodash.snakecase": "^4.1.1",
-    "lodash.camelcase": "^4.1.1",
-    "ramda": "^0.24.1",
-    "redux-actions": "^2.0.3"
+    "ramda": "0.24.1",
```

Comment:
Should we try updating this too? Or too much for this PR?

Reply:
I updated it and it broke stuff so I said nahhh, haha

Reply:
but we should in the future. i'll file an issue here so we keep track of it.

Reply:
filed #29